src/components/NcAppContent/NcAppContent.vue

```diff
@@ -43,6 +43,13 @@ all the children of the NcAppContent component
 	</NcAppContent>
 </template>
 ```
+```vue
+<template>
+	<NcAppContent layout-mode="no-split">
+		<h2>Single-column main content, full width list</h2>
+	</NcAppContent>
+</template>
+```
```

Comment:
`no-split` doesn't make sense in the example without the list, because `layout` is only used for `list + content`